core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1792,7 +1792,9 @@ public void delete(AlluxioURI path, DeleteContext context)
             throw e;
           }
         }
-        mMountTable.checkUnderWritableMountPoint(path);
+        if (!context.getOptions().getAlluxioOnly()) {
```

Comment:
Pls leave a comment here
```
// If the mount point is read only, we allow removing the in-Alluxio metadata and data
// in order to load it from the UFS again.
// This can happen if Alluxio is out-of-sync with the UFS.
```